projects/storefrontlib/src/cms-components/cart/cart-shared/cart-item-list/cart-item-list.component.ts

```diff
@@ -14,6 +14,9 @@ export class CartItemListComponent implements OnInit {
   @Input()
   hasHeader = true;
 
+  @Input()
+  promotionLocation: PromotionLocation = PromotionLocation.Cart;
```

Comment:
This assumes more `PromotionLocation.ActiveCart` yup? Because cart-item-list could possibly be used for other carts, not only the active one.